src/java/picard/analysis/artifacts/ConvertSequencingArtifactToOxoG.java

```diff
@@ -20,12 +20,42 @@
 import java.util.Set;
 
 @CommandLineProgramProperties(
-        usage = ConvertSequencingArtifactToOxoG.USAGE,
-        usageShort = ConvertSequencingArtifactToOxoG.USAGE,
+        usage = "Converts all sequencing artifact errors including all bait-bias and pre-adapter artifacts to " +
+                "8-oxoguanine artifacts <br /><br />." +
+                "Oxidation of guanine to 8-oxoguanine can occur during library preparation of genomic DNA via a " +
+                "combination of heat, shearing, and metal contaminates in a sample (doi: 10.1093/nar/gks1443).  It is " +
+                "one of the most common artifacts associated with genomic library preparation." +
+                "In DNA, 8-oxoguanine can pair with " +
+                "either cytosine or adenine, ultimately leading to G→T transversion mutations during PCR amplification.  " +
+                "The variants (C→A)/(G→T) tend to occur in specific sequence contexts e.g. CCG→CAG" +
+                "(doi:10.1093/nar/gks1443).  Although occurring at relatively low frequencies," +
+                " these artifacts can have profound impacts on variant calling fidelity (doi:10.1093/nar/gks1443).  " +
+                "<br /><br />This tool calculates the Phred-scaled probability that an alternate base" +
```

Comment:
Yes.  I wasn't sure about this one.  I need to find someone who can validate or correct...